lib/plugin/css-variables-collector.js

```diff
@@ -27,27 +28,41 @@ CSSVariablesCollectorPlugin.prototype = {
 		return this.ruleStack.length > 0 && !this.ruleStack[this.ruleStack.length - 1].variable;
 	},
 
-	_isInGlobalOrBaseImport() {
-		return this.config.libName !== "sap.ui.core" && this.importStack.filter((importNode) => {
-			return /\/(?:global|base)\.less$/.test(importNode.importedFilename);
-		}).length > 0;
+	_isVarInLibrary({varName, filename}) {
+		// for libraries we check that the file is within the libraries theme path
+		// in all other cases with no filename (indicates calculated variables)
+		// or in case of variables in standalone less files we just include them!
+		const include = !filename ||
+			(this.config.libPath ? filename.startsWith(`/resources/${this.config.libPath}/themes/`) : true);
+		return include;
 	},
 
 	_isRelevant() {
 		return !this._isInMixinOrParen() && this._isVarInRule();
 	},
 
-	toLessVariables() {
+	toLessVariables(varsOverride) {
+		const vars = {};
+		Object.keys(this.vars).forEach((key) => {
+			const override = this.vars[key].updateAfterEval && varsOverride[key] !== undefined;
+			if (override && process && process.env && process.env.LESS_OPENUI5_CSSVARS_REPORT_OVERRIDE) {
```

Comment:
console log only with ENV variable set (to be on the safe side for eventual usage in browser checking process.env to be available first)

Reply:
For that I think you would need to to a `typeof process` check, as just using `process` will result in a ReferenceError.